packages/core/types/Fields.ts

```diff
@@ -107,7 +110,10 @@ export type ExternalField<Props extends any = { [key: string]: any }> =
     }) => Promise<any[] | null>;
     mapProp?: (value: any) => Props;
     mapRow?: (value: any) => Record<string, string | number | ReactElement>;
-    getItemSummary?: (item: NotUndefined<Props>, index?: number) => string;
+    getItemSummary?: (
+      item: NotUndefined<Props>,
+      index?: number
+    ) => string | ReactNode;
```

Comment:
```suggestion
    ) => ReactNode;
```
ReactNode includes strings, numbers, or anything that can be used as React children.